src/ngx_http_tnt_version.h

```diff
@@ -34,6 +34,6 @@
 #ifndef NGX_HTTP_TNT_VERSION_H
 #define NGX_HTTP_TNT_VERSION_H 1
 
-#define NGX_HTTP_TNT_MODULE_VERSION_STRING "v2.7.2"
+#define NGX_HTTP_TNT_MODULE_VERSION_STRING "v2.7.3"
```

Comment:
I'm not sure about right release process here, but I find it more clear, when we do it separately from other changes. To be discussed.

Reply:
Dropped.